pkg/tests/non-dependant/yaml/smcp_v2.1.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: {{ .Name }}
+spec:
+  version: v2.1
+  tracing:
+    type: Jaeger
+    sampling: 10000
+  policy:
+    type: Istiod
+  addons:
+    grafana:
+      enabled: true
+    jaeger:
+      install:
+        storage:
+          type: Memory
+    kiali:
+      enabled: true
+    prometheus:
+      enabled: true
+  telemetry:
+    type: Istiod
+  {{ if eq .ROSA "true"}} 
+  security:
+    identity:
+      type: ThirdParty
+  {{ end }}
```

Comment:
Why duplicate these files? Code duplication always causes problems eventually.

Reply:
We need an agreement about where yaml templates will be added. Otherwise, yaml files or testdata may still get duplicated.

Reply:
If a file is only used by a single test, it should be located close to the test. It doesn't make sense to put all the files in a single global directory (with or without subdirectories), as that just makes everything disorganized.

Reply:
There are some files that are used by several tests like the smcp yaml file, I added the operators yaml too because if we are going to run in a moment by package we are going to need those yaml I think

Reply:
Fixed